app/controllers/api/v1/application_controller.rb

```diff
@@ -10,8 +10,14 @@ class ApplicationController < ActionController::API
 
       before_action :doorkeeper_authorize!
 
+      rescue_from ActionController::ParameterMissing, with: :handle_parameter_missing
+
       private
 
+      def handle_parameter_missing(exception)
+        render json: { error: exception.message }, status: :bad_request
```

Comment:
You can use `render_errors`
```suggestion
render_errors(
          details: [exception.message],
          status: :bad_request
        )
```
or add this rescue in `Rescuable.rb` as well as `handle_parameter_missing` to group the rescues we have.
```
      included do
        rescue_from ActiveRecord::RecordNotFound, with: :render_not_found_error
      rescue_from ActionController::ParameterMissing, with: :handle_parameter_missing
      end
```